packages/components/src/Combobox/components/ComboboxContent/ComboboxContent.test.tsx

```diff
@@ -83,6 +92,7 @@ function renderComboboxContent(
       shouldScroll={{ current: false }}
       open={open}
       selected={selected}
+      searchValue=""
```

Comment:
hmm I probably should have made this `searchValue={searchValue}` rather than hardcoding it to an empty string

it doesn't really matter, the tests still pass, but it is silly